src/app/(proper_react)/redesign/(public)/LandingView.test.tsx

```diff
@@ -24,9 +24,24 @@ import Meta, {
   LandingUsScanLimit,
 } from "./LandingView.stories";
 
-jest.mock("next-auth/react");
+jest.mock("next-auth/react", () => {
+  return {
+    signIn: jest.fn(),
+    useSession: jest.fn(() => {
+      return {};
+    }),
+  };
+});
 jest.mock("../../../hooks/useTelemetry");
 
+beforeEach(() => {
+  // For reasons that are unclear to me, the mock implementation defind in the
+  // call to `jest.mock` above forgets the implementation. I've spent way too
+  // long debugging that already, so I'm settling for this :(
+  const mockedUseSession = useSession as jest.Mock;
+  mockedUseSession.mockReturnValue({});
```

Comment:
There's a call to `useSession` [here](https://github.com/mozilla/blurts-server/pull/3990/files#diff-73a4c0c3e9533af14ca2a89a97b1e4c8b67e0a4c2b2120b35f472ffc149eec2dR13), which [expects](https://github.com/mozilla/blurts-server/pull/3990/files#diff-73a4c0c3e9533af14ca2a89a97b1e4c8b67e0a4c2b2120b35f472ffc149eec2dR15) the return value to be an object with optionally a `data` property on it.

We're mocking out `next-auth/react`, but if we don't specify a return value for the mocked `useSession`, it will return `undefined`, resulting in this error:

    Error: Uncaught [TypeError: Cannot read properties of undefined (reading 'data')]

Theoretically, [these lines](https://github.com/mozilla/blurts-server/pull/3990/files#diff-ff0a0f27106b116a3c2d0c1216460f3c6d5fd688772ddb7944ec4acff4df31a6R30-R32) should be sufficient to let the `useSession` mock return an actual object (albeit without `.data`).

But for some reason that I can't figure out, that is insufficient :/ And so these lines make that mock return an empty object again before every test...